Project.toml

```diff
@@ -0,0 +1,14 @@
+name = "AdvancedPS"
+uuid = "9c00393c-822d-4781-8df5-4c3c33f9866d"
+authors = ["JS Denain <denain.js@gmail.com>"]
+version = "0.1.0"
+
+[deps]
+AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+DynamicPPL = "366bfd00-2699-11ea-058f-f148b4cae6d8"
+Libtask = "6f1fad26-d15e-5dc8-ae53-837a1d7b8c9f"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Turing = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
```

Comment:
The Manifest.toml file should be removed and Project.toml should be cleaned: test dependencies have to be moved to a separate section (e.g., this package shouldn't depend on Test or Turing, and probably also not on DynamicPPL), and compatibility bounds have to be specified.

Reply:
what do you mean by 'moved to a separated section'?

Reply:
Yes, exactly :+1: